### PR TITLE
clean up. Gradle CI will deploy snapshots, bintray as some private artifacts

### PR DESCRIPTION
**Pull request by stephanenicolas**
Fix issues #88, #89, #91

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### Snapshots
+Snapshots of the lib are created for every change on master.
+They are accessible on gradle's repo:
+```gradle
+repositories {
+    maven { url 'https://repo.gradle.org/gradle/libs-snapshots' }
+}
+```
+
+### Release artifacts before 1.0.0
```
Comment (oehme) [inline RELEASING.md]: Please stick with the Gradle repo and tell us when you need a release.

**Comment by stephanenicolas** [comment]
Where do we remove the ability to use Gradle repo for releases ?
On our side, it has proven handy to be able to get stable artifacts on
bintray.

**Comment by oehme** [comment]
This PR removes maven-push.gradle which is responsible for publishing to our repo.

Again, please just let us know when you need a stable artifact. Let's not confuse people by putting incap in several different places, using different processes and different versions.

**Comment by oehme** [comment]
Nevermind the first part, I see we no longer use maven-push.gradle. My other point still stands though.

**Comment by stephanenicolas** [comment]
@oehme we will definitely kill the bintray repo when it's not used anymore. 

For now we use it to store incap's binaries, and also the butterknife binaries after we fork it. We also need a place for them as well so that we can test our sample apps and get stability while keeping on developing them.

**Comment by oehme** [comment]
If that repo gets deleted later, I'm fine with this.

**Comment by stephanenicolas** [comment]
100% guaranteed, I don't want to spread dev binaries neither. I would be happy to give you access to the repo so you can delete them too.

**Pull request merged**